setup.py

```diff
@@ -21,7 +21,7 @@
     long_description_content_type='text/markdown',
     keywords=['Security', 'SSH', 'Secret Keys', 'SysAdmin'],
     install_requires=required,
-    python_requires='>=3.4',
```

Comment:
Why did we bump python version ?

Reply:
See: https://github.com/valayDave/tell-me-your-secrets/pull/17#discussion_r488693949

Do you have Python 3.4 available to test?

Reply:
I could try with pyenv...

Reply:
```
root@2f3d0741b011:/code# python
Python 3.4.10 (default, Mar 20 2019, 00:52:37) 
[GCC 6.3.0 20170516] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import re
>>> m = re.match('(?i:(facebook|fb)(.{0,20})?)[''\"][0-9a-f]{32}[''\"]', 'faceBOOK="aaaabbbbccccddddaaaabbbbccccdddd"')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.4/re.py", line 160, in match
    return _compile(pattern, flags).match(string)
  File "/usr/local/lib/python3.4/re.py", line 294, in _compile
    p = sre_compile.compile(pattern, flags)
  File "/usr/local/lib/python3.4/sre_compile.py", line 568, in compile
    p = sre_parse.parse(p, flags)
  File "/usr/local/lib/python3.4/sre_parse.py", line 780, in parse
    p = _parse_sub(source, pattern, 0)
  File "/usr/local/lib/python3.4/sre_parse.py", line 377, in _parse_sub
    itemsappend(_parse(source, state))
  File "/usr/local/lib/python3.4/sre_parse.py", line 741, in _parse
    raise error("unknown extension")
sre_constants.error: unknown extension
```

yeah it blows up.. even installing deps don't work

```
root@2f3d0741b011:/code# python --version
Python 3.4.10
root@2f3d0741b011:/code# pip install -r requirements.txt 
DEPRECATION: Python 3.4 support has been deprecated. pip 19.1 will be the last one supporting it. Please upgrade your Python as Python 3.4 won't be maintained after March 2019 (cf PEP 429).
Collecting pyyaml (from -r requirements.txt (line 1))
  Using cached https://files.pythonhosted.org/packages/64/c2/b80047c7ac2478f9501676c988a5411ed5572f35d1beff9cae07d321512c/PyYAML-5.3.1.tar.gz
PyYAML requires Python '>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*' but the running Python is 3.4.10
You are using pip version 19.0.3, however version 19.1.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```

Reply:
p.s. Python 3.4 is EOL

Reply:
Extended CI test suite to cover versions 

https://github.com/valayDave/tell-me-your-secrets/pull/25/files

From manual testing things don't build on 3.4 or 3.5 - are you on any Python version that low to verify?

Reply:
Yeah even on CI 3.4 and 3.5 fail completely - I looked and pyyaml doesn't install with the unbounded versions specified.